### PR TITLE
Overall review of Core docs 2.1.6 - Closes #674

### LINKED ISSUE #674: Overall review of Core docs 2.1.6
Overall review of tone of voice, style, grammar and typos, etc. after structural changes and link fixes.

Parent issue #665

### PR DESCRIPTION
**Pull request by chris529**
Overall review of structure, tone of voice, style, grammar, etc, etc.

Closes #674

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -49,10 +49,10 @@ Furthermore, this also applies to every additional machine that is required to r
 Setting up the Lisk Core provides the user with the following options:
 
 * The ability to connect to a <<networks, network>> and communicate with other nodes in the network.
-* To have full control to xref:{url_config}[configure] the Lisk Core to specific and custom requirements.
-* To create your own <<snapshots, snapshot>> of the blockchain.
-* Use Lisk Core to perform actions on the Lisk blockchain, e.g. with {url_lisk_desktop}[Lisk Desktop].
-* xref:{url_config_forging}[Forge] new blocks, (for active delegates).
+* The full control to xref:{url_config}[configure] the Lisk Core to specific and custom requirements.
+* The possibility to create your own <<snapshots, snapshot>> of the blockchain.
+* The ability to use Lisk Core to perform actions on the Lisk blockchain, e.g. with {url_lisk_desktop}[Lisk Desktop].
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* The ability to use a self-controlled node to perform actions on the Lisk blockchain, e.g. with {url_lisk_desktop}[Lisk Desktop].
```

File: modules/ROOT/pages/management/logs.adoc [hunk 2]
```diff
@@ -32,7 +32,7 @@ This also requires the operator to investigate the problem, although it does not
 
 |Debug(20) |Anything else, i.e. too verbose to be included in `info` level.
 
-|Trace(10) |Logging from external libraries used by your node, or else very detailed application logging.
+|Trace(10) |Logging from external libraries used by your node, or otherwise very detailed application logging.
```
Comment (Tschakki) [inline modules/ROOT/pages/management/logs.adoc]:
```suggestion
|Trace(10) |Logging from external libraries used by your node, or other very detailed application logging.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looks good to me, just 2 minor points

**Comment by chris529** [comment]
Thanks Mona! Well spotted! 👍

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**